ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -59,41 +61,41 @@ internal class PaywallViewModelImpl(
     preview: Boolean = false,
 ) : ViewModel(), PaywallViewModel {
 
-    private val variableDataProvider = VariableDataProvider(applicationContext, preview)
+    val variableDataProvider = VariableDataProvider(applicationContext, preview)
 
     override val state: StateFlow<PaywallViewState>
         get() = _state.asStateFlow()
+
+    val mode: PaywallViewMode
```

Comment:
Same here, I think this is only used inside this class?

Reply:
Fixed

Reply:
I was missing `fileprivate` 😅 but I just moved `calculateState` inside